golem-worker-executor-base/src/services/rdbms/tests.rs

```diff
@@ -2846,6 +2847,147 @@ fn test_rdbms_pool_key_masked_address() {
     check!(key.masked_address() == "postgres://user:*****@localhost:5432?abc=xyz&secret=*****");
 }
 
+#[test]
+async fn mysql_par_test(mysql: &DockerMysqlRdb, rdbms_service: &RdbmsServiceDefault) {
+    let db_address = mysql.public_connection_string();
+    let rdbms = rdbms_service.mysql();
+    let mut db_addresses = create_test_databases(rdbms.clone(), &db_address, 3, |db_name| {
```

Comment:
As this is using the same db address, the pool will not create independent connections.
Instead, it will reuse the same connection for all 3 of the tests.

What exactly do you want to test here?
* That multiple workers can be connected to the same database? If yes, this is already implicitly tested through the various tests in this module getting executed concurrently. I would not say a dedicated test for this is unneccessary.

* That multiple workers can be connecto to multiple databases? If yes, we need to have different connection strings for them to avoid sharing in the pool. The way I would propose this is to start the same container using multiple aliases, i.e.
`docker create --network-alias db_1 --network-alias db_2 ...` and run tests against those.

Reply:
probably variable names are not best,

but those are db connections urls  

```
"mysql://root:mysql@localhost:55000/test_db_0", "mysql://root:mysql@localhost:55000/test_db_1", "mysql://root:mysql@localhost:55000/test_db_2", "mysql://root:mysql@localhost:55000/mysql"
```
pools are created for url, which may have connection to specific underlying database, with specific user, or additional custom attributes

test are aiming to test if `RdbmsServiceDefault` working properly with multiple pools accessing by multiple workers

Reply:
missed the db_name part in the connection strings.

This looks nice then, lgtm